ViAn/GUI/TreeItems/tagframeitem.cpp

```diff
@@ -0,0 +1,20 @@
+#include "tagframeitem.h"
+
+TagFrameItem::TagFrameItem(int frame) : TreeItem(TAG_FRAME_ITEM) {
+    m_frame = frame;
+    setText(0, QString::number(m_frame));
+}
+
+void TagFrameItem::set_state(TagFrame* t_frame) {
+    m_t_frame = t_frame;
+}
+
+void TagFrameItem::remove(){}
+
+void TagFrameItem::rename() {
+    // TODO
```

Comment:
Is this fixed in any of the other branches? Just so that we won't forget

Reply:
Currently not fixed, but also not sure if wanted. The tag frames have the frame number as name right now. Do we wanna give the user the option to change that name? If we do, it'll come in a later pull request.

Reply:
Ah, didn't really think about that. Frame number should be enough then.

Reply:
Can also say that right now the user can change the name of them but the name should not be saved. We might wanna lock the name field so it can't be changed especially if we don't save the new name.